src/Altinn.App.Core/Internal/App/FrontendFeatures.cs

```diff
@@ -10,9 +13,14 @@ public class FrontendFeatures : IFrontendFeatures
         /// <summary>
         /// Default implementation of IFrontendFeatures
         /// </summary>
-        public FrontendFeatures()
+        public FrontendFeatures(IFeatureManager featureManager)
         {
             features.Add("footer", true);
+
+            if (featureManager.IsEnabledAsync(FeatureFlags.JsonObjectInDataResponse).Result)
+            {
+                features.Add("json_object_in_data_response", true);
```

Comment:
Nope, think I did this because I thought I saw you did it somewhere else and that this was what you agreed upon. I like camelCase :-D 

Good point - I'll change.